docs/running.rst

```diff
@@ -31,7 +31,9 @@ Currently there are only two available actions:
 Listing of the regression tests
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To retrieve a listing of the selected checks, you must specify the ``-l`` or ``--list`` options.
+There are two ways to retrieve a listing of the selected checks. One is by specifying the ``-l`` or ``--list`` option.
```

Comment:
The guideline in the documentation is to put a single sentence per line, so as to facilitate the diffs and avoid conflicts. There is now line limit in the documentation.